### PR TITLE
Adding config for ScriptsServer node type

### PR DESCRIPTION
**Pull request by chrisgilbert**
Some additions for a ScriptServer type and a couple of new IAM groups needed.

### CODE REVIEW

File: tyr/servers/scripts/server.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+        super(ScriptsServer, self).__init__(group, server_type, instance_type,
+                                            environment, ami, region, role,
+                                            keypair, availability_zone,
+                                            security_groups, block_devices,
+                                            chef_path, subnet_id, dns_zones)
+
+    def configure(self):
+        if not self.instance_type:
+            if self.environment == 'prod':
```
Comment (citruspi) [inline tyr/servers/scripts/server.py]: Why check the environment if the same instance type is used regardless of the environment?

File: tyr/servers/scripts/server.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+                self.instance_type = 't2.micro'
+            else:
+                self.instance_type = 't2.micro'
+        super(ScriptsServer, self).configure()
+
+        self.resolve_iam_role()
```
Comment (citruspi) [inline tyr/servers/scripts/server.py]: This shouldn't be needed - it should be called as part of `configure()`.

File: tyr/servers/scripts/server.py (hunk 1 above)
Comment (chrisgilbert) [inline tyr/servers/scripts/server.py]: Good point..

File: tyr/servers/scripts/server.py (hunk 2 above)
Comment (chrisgilbert) [inline tyr/servers/scripts/server.py]: Yep, I see that, I will remove it.

**Pull request merged**